src/less/tables.less

```diff
@@ -72,7 +72,7 @@
             text-align: left;
         }
 
-        &.table-default__cell-center { /* [2], [3] */
```

Comment:
This is a breaking change. Also, seems like this may belong in the  `#TableCell` section as well, so it's obvious it exists, and can be used. 

I recommend/desire keeping the existing `table-default__cell-center` so it doesn't break existing applications.